debezium-testing/debezium-testing-testcontainers/src/main/java/io/debezium/testing/testcontainers/ApicurioInfrastructure.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.debezium.testing.testcontainers;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+/**
+ * Apicurio Testcontainers infrastructure handling.
+ */
+
+public class ApicurioInfrastructure extends GenericContainer<ApicurioInfrastructure> {
+
+    protected static final String DEBEZIUM_VERSION = "1.3.0.Final";
+    protected static final String APICURIO_VERSION = "1.3.2.Final";
+    private final GenericContainer<?> apicurioContainer;
+    private final ImageFromDockerfile apicurioContainerImage;
+
+    public static ConnectInfrastructure CONNECT_INFRASTRUCTURE = ConnectInfrastructure.getDefaultConnectInfrastructure();
+
+    private final String connectContainer;
+
+    private ApicurioInfrastructure() {
+        connectContainer = CONNECT_INFRASTRUCTURE.getConnectContainer();
+        apicurioContainerImage = new ImageFromDockerfile()
+                .withDockerfileFromBuilder(builder -> builder
+                        .from(connectContainer)
+                        .env("KAFKA_CONNECT_DEBEZIUM_DIR", "$KAFKA_CONNECT_PLUGINS_DIR/debezium-connector-postgres")
+                        .env("APICURIO_VERSION", APICURIO_VERSION)
+                        .run("cd $KAFKA_CONNECT_DEBEZIUM_DIR && curl https://repo1.maven.org/maven2/io/apicurio/apicurio-registry-distro-connect-converter/$APICURIO_VERSION/apicurio-registry-distro-connect-converter-$APICURIO_VERSION-converter.tar.gz | tar xzv")
+                        .build());
```

Comment:
That's not needed actually (see my other comment below), current Debezium versions contain those converters already, it's only a matter of enabling them by means of an env var. I.e. this class here should not be needed, and instead a test wishing to use Connect/DBZ with the Apicurio converters would do it like so:

```
public static DebeziumContainer debeziumContainer = new DebeziumContainer()
            .withNetwork(network)
            .withKafka(kafkaContainer)
            .withLogConsumer(new Slf4jLogConsumer(LOGGER))
            .enableApicurioConverters()
            .dependsOn(kafkaContainer);
```